mizar/get_interface.py

```diff
@@ -0,0 +1,11 @@
+import os
+import logging
+logger = logging.getLogger()
+
+def get_itf():
+    default_itf = os.popen("route | grep '^default' | grep -o '[^ ]*$'").read().split('\n')[0]
+    if "NET_ITF" in os.environ:
+        return os.getenv("NET_ITF")
```

Comment:
Can you change the name from "NET_ITF" to "MIZAR_ITF"? Because each machine has many environment variables, we want distinguish mizar specific env var from the reset

Reply:
Updated the environment variable name as per the suggestion.